src/main/java/ui/components/NavigationBar.java

```diff
@@ -0,0 +1,22 @@
+package main.java.ui.components;
+
+import org.kordamp.ikonli.fontawesome5.FontAwesomeSolid;
+import org.kordamp.ikonli.swing.FontIcon;
+
+import javax.swing.*;
+
+public class NavigationBar extends JPanel {
```

Comment:
Doc

Reply:
@Gord310, I will add further doc when implementing the hole navbar